### ISSUE TITLE
The keyboard user is unable to Zoom in and Zoom out using the Command+Plus/Minus keys as mentioned in the menu bar of the application.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#294]
### Pre-requisites:
Launch the Visual Studio for Mac and Sign-in with Enterprise account.

Create projects:

1. Go to Visual Studio-> File-> New Solution. -> Multiplatform-> App-> Xamarin.Forms-> Blank Forms App.
2. Hit on Next button to create the project.
3. Hit on Debug(Play)/Build button to build/deploy the solution/project.
4. As per the choice/requirements, Change the Device platform like iOS/ Android and iPhone X iOS 11.3 etc.
5. Hit on Inspector icon button (i.e., in the top toolbar buttons of the VS) to inspect the project.
6. On hitting on inspector button, Xamarin Inspector is Launched.

### Repro-Steps:
1. Follow the Pre-requisites to Launch the Xamarin Inspector from Visual Studio.
2. Navigate to the Top menu bar-> View-> Zoom In/Zoom Out.
3. Try to zoom the application using mentioned key in the menu bar of the application as below:
Zoom In: Command+ Plus key
Zoom Out: Command+ Minus key
Actual Size: Command + Zero

### Expected Behavior
The below mentioned key should work as it is as mentioned in the menu bar of the Xamarin Inspector application:
Zoom In: Command+ Plus key
Zoom Out: Command+ Minus key
Actual Size: Command + Zero
### Actual Behavior
The keyboard user is unable to Zoom in and Zoom out using the Command+Plus/Minus keys as mentioned in the menu bar of the application.
### User Impact
As per the keys which are mentioned in the application's menubar, user might expect the same behavior, but if it won't follow the same expectation, the user might be confused.
The user won't be able to zoom in and zoom out the application. 

![kb10_menubar-unabletozoomin-outusingmentionedcommand](https://user-images.githubusercontent.com/22956130/39987957-c1b71bec-5783-11e8-8b61-5614a2e10170.png)

### Environment
Visual Studio for Mac Enterprise [Version: 7.5 Preview (7.5 build 1222)]
Xamarin Inspector [Version: 1.4 (master/b3f92f9)]

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Inspector-MAC
Wipro
MAS2.1.1

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1830

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1830
